app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheet.kt

```diff
@@ -99,8 +99,8 @@ class BloodPressureEntrySheet : BottomSheetActivity(), BloodPressureEntryUi {
 
   private val delegate by unsafeLazy {
     val defaultModel = when (val openAs = intent.extras!!.getParcelable(KEY_OPEN_AS) as OpenAs) {
-      is New -> BloodPressureEntryModel.newBloodPressureEntry(openAs.patientUuid)
-      is Update -> BloodPressureEntryModel.updateBloodPressureEntry(openAs.bpUuid)
+      is New -> BloodPressureEntryModel.newBloodPressureEntry(New(openAs.patientUuid))
```

Comment:
What I meant was that we could just do

```kotlin
val openAs = intent.extras!!.getParcelable(KEY_OPEN_AS) as OpenAs
val defaultModel = BloodPressureEntryModel(openAs)
```

Reply:
The work of deciding the right `OpenAs` is already decided by the previous screen. This screen shouldn't have to worry about it.

Reply:
Since I am migrating 1 feature at a time, I prefer to break into one execution path at once. We can unify this in the final PR.

Reply:
Alright.